sspl_test/common.py

```diff
@@ -31,19 +31,13 @@
 import psutil
 import subprocess
 from threading import Thread
-from sspl_test.default import *
+from default import *
 
-from sspl_test.framework.utils.service_logging import init_logging
-from sspl_test.framework.utils.service_logging import logger
-
-# Adding sspl and sspl_test path
-test_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-os.sys.path.append(os.path.join(test_path))
-os.sys.path.append(os.path.join(test_path + "/../"))
-
-from sspl_test.rabbitmq.rabbitmq_ingress_processor_tests import RabbitMQingressProcessorTests
-from sspl_test.rabbitmq.rabbitmq_egress_processor import RabbitMQegressProcessor
-from sspl_test.framework.utils.conf_utils import Conf, SSPL_TEST_CONF, GLOBAL_CONF
+from framework.utils.service_logging import init_logging
```

Comment:
this wont be confused with sspl.framework ?

Reply:
Unfortunately, the code folder directory name is the same as the SSPL module core code.
But above code imported from inside the sspl_test module so I don't think so it will be confused.

If you think it is confusing then we can put all the core directories to one specific folder (sspl_test) and will import them from there. But I don't think so that will be convinced or proper way.

Reply:
> above code imported from inside the sspl_test module so I don't think so it will be confused.

yeah we can relate that way, not a problem. I was thinking functionally code wont import from sspl.framework, but code hierarchy would take care. should be fine